ADRs/RH/SIG-SRE/ADR-00002 Personas related to Managed Services.md

```diff
@@ -0,0 +1,119 @@
+# ADR-00002 Personas related to Managed Services
+
+Authors: Craig Robinson
+
+
+## Status
+
+Draft
+
+
+## Problem Statement
+
+There is content within SIG-SRE that is being produced that depends upon the defintion of a "persona" as it is related to Managed Services. For example, as a consumer of SLO's, I want to be able to consume the data in a relevant and consistent way in accordance with my "persona". Similarly, there is a RACI chart which has persona as a key datapoint.
+
+Therefore, to assist in the production of this other content, it is necessary to first define the persona. Some examples are (but not limited to): SRE, Engineer, Product Owner, Business, Executive.
+
+
+## Goals
+
+This ADR hopes to resolve the problem statement by identifying and documenting the key personas in a summary format. The output will be a document which will be accessible by other assets.
+
+
+## Non-goals
+
+* This ADR does not aim at capturing every form or type of persona. It is purposefully generalised, utilising the most frequent persona types as experienced within the Red Hat 'Managed Services' context. The reason is that every organisation calls their personas by different names (and they will inevitably have different role descriptions). Nevertheless, this document hopes to capture the main types in a translatable way.
```

Comment:
IMO, I think this should stay: mainly to circumvent future objections like: "where did you come up with the names of those personas, because my company does not call them that". :) At least we are being explicit? WDYT @jeremyeder ?

Reply:
OK, fair.